src/golang/lib/job/spec.go

```diff
@@ -174,6 +175,15 @@ type DiscoverSpec struct {
 	OutputContentPath string              `json:"output_content_path"  yaml:"output_content_path"`
 }
 
+type CompileAirflowSpec struct {
+	BasePythonSpec
+	OutputContentPath string              `json:"output_content_path"  yaml:"output_content_path"`
+	DagId             string              `json:"dag_id"  yaml:"dag_id"`
+	CronSchedule      string              `json:"cron_schedule"  yaml:"cron_schedule"`
+	TaskSpecs         map[string]Spec     `json:"task_specs"  yaml:"task_specs"`
```

Comment:
does any task spec fits here? If not we should add some comments on restrictions, and also do some type checking in the constructor

Reply:
Any operator spec works. I'll add some checking.